docs/source/actions.rst

```diff
@@ -196,13 +196,19 @@ List of recipients of the message. It could be a channel or an user.
 
 SPIDERMON_SLACK_SENDER_NAME
 ---------------------------
+Name of app/bot/user created for serving notifications on desired channels.
```

Comment:
Is this the account nickname (account ID?) or the display name? I wonder if this should me more clear from the documentation.

Some :lipstick:  aesthetic notes:
- You should probably leave a blank line between this sentence and the header underlining.
- Would it make sense to replace ‘app/bot/user’ with ‘Slack account’?
- Careful with [article omission](https://www.englishgrammar.org/omission-articles/).

Reply:
Yes, this is the username of the Slack app that needs to be created for Slack Action. I will modify the same. 

> Would it make sense to replace ‘app/bot/user’ with ‘Slack account’?
Reading more into it now, I think `app/bot` would be a better choice since `Slack API` is used to creating `Slack Apps`. Those `Slack Apps` are clalled bots. [Reference #1](https://api.slack.com/bot-users#bots-overview) [Refer #2](https://api.slack.com/slack-apps)

In reference to a test bot I created a while ago:
![Screenshot_2019-05-03_21-08-11](https://user-images.githubusercontent.com/22801822/57148665-b7cc4000-6de7-11e9-9bf4-41c17c50e6b5.png)

This is how it looks like
![Screenshot_2019-05-03_21-09-54](https://user-images.githubusercontent.com/22801822/57148727-dcc0b300-6de7-11e9-8812-abbe257f7496.png)